client/my-sites/hosting/cache-card/index.js

```diff
@@ -139,6 +146,18 @@ export const CacheCard = ( {
 			</div>
 		);
 	};
+
+	const edgeCacheToggleDescription = isEdgeCacheEligible
+		? translate( 'Enable edge caching for faster content delivery.' )
+		: translate(
+				'Edge cache can only be enabled for public sites. {{a}}Confirm privacy settings{{/a}}.',
+				{
+					components: {
+						a: <a href={ '/settings/general/' + siteSlug + '#site-privacy-settings' } />,
+					},
+				}
+		  );
```

Comment:
@javierarce What do you think about this conditional toggle description text?

Reply:
"Confirm" is not very accurate. I think it would be better to say: "Update your privacy settings" or "Change your privacy settings".

Reply:
@javierarce The intent with "Confirm" is that you would only click the link if you believed you saw the disabled toggle in error. "Update" and "Change" both direct the user to modify their settings. What language would you suggest for "you only need to click this if you think it's unexpected"?

Reply:
In that case, I would probably say "review" instead. But I think the other options would be ok too, since we are just offering the user a shortcut to change the privacy.

Reply:
Sounds good, updated to "Review" with b0b2e4d4ec